src/index.js

```diff
@@ -1,6 +1,7 @@
 var multiaddr = require('multiaddr')
 var getConfig = require('./config')
 var getRequestAPI = require('./request-api')
+var request = require('request')
```

Comment:
@diasdavid @VictorBjelkholm no need for all that, all tests are passing in the browser, request is 100% browser compatible :)